scania-digital-design-system/sdds
# Create migration page

**Describe pull-request**  
Created a page for the migration where we can log breaking changes.

**Solving issue**  
Fixes: AB#2494

**How to test**  
1. Go to storybook link below.
2. Check in Intro -> Migration

**Screenshots**  
<img width="1509" alt="Screenshot 2022-11-04 at 12 57 43" src="https://user-images.githubusercontent.com/62651103/199967244-4a985442-3f11-433f-a754-52bb8ff35a98.png">


**Additional context**  
-
